scripts/airflow3/requirements.txt

```diff
@@ -3,8 +3,9 @@ apache-airflow-task-sdk
 apache-airflow-providers-standard
 apache-airflow-providers-fab
 apache-airflow-providers-cncf-kubernetes
+# apache-airflow-providers-amazon[s3fs]
```

Comment:
Is there any reason why this is commented out?

Reply:
Installing apache-airflow-providers-amazon[s3fs] and apache-airflow-providers-google together causes a dependency resolution conflict.

Reply:
Could you add a comment, please?